resources/Dockerfile

```diff
@@ -125,6 +125,35 @@ RUN if [ "$TARGETARCH" = "amd64" ]; then \
        && cp -r ./scripts/rpc /usr/local/bin/spdk-nvme \
        && cd .. && rm -rf spdk; fi
 
+# install swtpm only for x86_64 arch
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+    apt-get update \
+    && apt-get -yq upgrade \
+    && DEBIAN_FRONTEND=noninteractive apt-get  install -yq gawk libssl-dev libtool pkg-config \
+    expect gnutls-dev gnutls-bin \
+    libfuse-dev libglib2.0-dev libjson-glib-dev libgmp-dev libtasn1-dev \
+    socat python3-twisted net-tools softhsm2 libseccomp-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
Can you please consolidate the dependencies to the "apt-get" step above from line 58 and also ensure no duplicate packages?

Reply:
Done!